src/main/frontend/src/components/App.js

```diff
@@ -3,20 +3,27 @@ import { Switch, Route } from "react-router-dom";
 import Navbar from './Navbar.js';
 import WordsContainer from './WordsContainer.js';
 import WordShow from './WordShow.js';
+import WordForm from './WordForm.js';
 
 function App() {
   return (
     <>
       <Navbar />
 
       <Switch>
-        <Route path="/words/:id">
+        <Route exact path="/words/new">
+          <WordForm />
+        </Route>
```

Comment:
Although this syntax is fine and functional, I would typically prefer to combine into one line as so:
```
<Route exact path="/words/new" component={WordForm} />
```